decidim/decidim
# Add possibility of overriding the tailwind.config.js

#### :tophat: What? Why?
At the europeean commission we need to add some customization in the font face and other elements, and, for that would be ok to add them in Tailwind . For instance, several things that we need to override : 

- Adjust font sizes 
- Add a new font family 
- Change some paddings 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to N/A

#### Testing
1. Create a development application 
2. create a file named `config/tailwind.config.js.erb` following the [template](https://github.com/decidim/decidim/blob/develop/decidim-core/lib/decidim/assets/tailwind/tailwind.config.js.erb)
3. Edit the newly created file ( ex change Font sizes )
4. See the generated tailwind config contains the values customized

:hearts: Thank you!
